building: add Windows quick start with ia32 (#162)
phoenix-rtos/phoenix-rtos-doc: building/README.md

Instructions in the `Building` and `Running system on targets` chapters have been verified for the Ubuntu
(20.04 and 22.04 versions) Linux distribution and macOS (tested on macOS Monterey 12.6.1), so this is the easiest way

> to start working with Phoenix-RTOS. Using Windows is possible with using Cygwin or Wsl or try to use Docker.

Reviewer: Update: We will not be adding instructions for Docker as previously discussed. Docker on Windows is primarily supported through WSL.

Instructions in the `Building` and `Running system on targets` chapters have been verified for the Ubuntu
(20.04 and 22.04 versions) Linux distribution and macOS (tested on macOS Monterey 12.6.1), so this is the easiest way

> to start working with Phoenix-RTOS. Using Windows is possible by using tools like `Cygwin` or `WSL`. The other way is to
> try to use Docker.

Reviewer 2: I think that we can resign from suggesting Docker as we don't have it described and using wsl is much simpler.

```suggestion
to start working with Phoenix-RTOS. Windows is also supported, by using `Cygwin` or `WSL`.
```
